two_factor/templates/two_factor/core/login.html

```diff
@@ -49,4 +49,35 @@ <h1>{% block title %}{% trans "Login" %}{% endblock %}</h1>
 
     {% include "two_factor/_wizard_actions.html" %}
   </form>
+
+  {{ form.media }}
+  {% if webauthn %}
+    <script>
```

Comment:
There should be no JS in HTML like this. It makes implementing CSP difficult as you can no longer tell the browser to block inline JS.

Reply:
No more inline JS